addons/mail/static/src/components/discuss_sidebar_item/discuss_sidebar_item.js

```diff
@@ -10,6 +10,7 @@ const useStore = require('mail/static/src/component_hooks/use_store/use_store.js
 const Dialog = require('web.Dialog');
 
 const { Component } = owl;
+const { isEventHandled } = require('mail/static/src/utils/utils.js');
```

Comment:
define just below `const useStore = ...` (and keep single empty line before `const Dialog = ...`)